### ISSUE TITLE
Vue is not interpolating template strings.

### ISSUE THREAD

**Issue by trusktr** [vuejs/vue#7295]
### Version
2.5.13

### Reproduction link
[https://codepen.io/trusktr/pen/EoVGge](https://codepen.io/trusktr/pen/EoVGge)

### Steps to reproduce
In that pen, you'll see errors in the console that look like this:

```
Error: Attribute must be a comma- or space-separated sequence of up to three numbers, for example "1 2.5 3". Yours was "`0 0 ${ n * 360/48   360/48/2 }`".
```

### What is expected?
There should not be these errors. I am expecting Vue to interpolate the template strings.

### What is actually happening?
These error happen because Vue is passing the literal template strings to the custom elements (the `<i-node>` elements), and those elements are throwing those errors.

Vue should be interpolating the strings.

**Comment by trusktr** [comment]
Hmmm, it doesn't seem to be the template strings. If I change most of them to arrays, it seems that Vue is not interpolating those either.

https://codepen.io/trusktr/pen/opLmyv?editors=1010

In the console, the custom elements (`<i-node>`) report that strings containing stuff like `[a,b,c+d]` were passed in, and this should not be the case because if Vue were interpolating the arrays, then the values passed into the custom elements would be of the form `"a, b, c"` rather than `"[a, b, c+d]"` because the when arrays are converted to string they show a comma-separated list of the values.

Why is Vue not interpolating the attributes?

**Comment by trusktr** [comment]
Hmmm, looks like I got it working:

https://codepen.io/trusktr/pen/BJzbBw

But I'm not sure why it was working before I made any changes when I was using Single File Components with Babel to compile them as opposed to having Vue compile the templates at runtime as in the Codepens.

I've honestly got no clue. 😃 

Maybe it has to do with this:

https://vuejsdevelopers.com/2017/09/17/vue-js-avoid-dom-templates/

But I'm not sure.
